docs/source/transforms.rst

```diff
@@ -162,9 +192,15 @@ Conversion
     :template: class.rst
 
     ToPILImage
+    v2.ToPILImage
+    v2.ToImagePIL
     ToTensor
+    v2.ToTensor
```

Comment:
Since it is deprecated, should we document it?

Reply:
It seems that `ToImageTensor` is missing from the list.

Reply:
> Since it is deprecated, should we document it?

Some thoughts here https://github.com/pytorch/vision/pull/7297/files#r1113244517

> It seems that ToImageTensor is missing from the list.

I did not add any new v2 transforms here, so that's normal